cli/src/commit_templater.rs

```diff
@@ -1671,3 +1730,83 @@ fn builtin_tree_diff_methods<'repo>() -> CommitTemplateBuildMethodFnMap<'repo, T
     // TODO: add files() or map() to support custom summary-like formatting?
     map
 }
+
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
nit: better to not implement `Eq, PartialEq` because it doesn't compare the signature.